Makefile

```diff
@@ -1,60 +1,67 @@
-VENV = venv
-PYTHON = $(VENV)/bin/python3
-PIP = $(VENV)/bin/pip
```

Comment:
This parameter didn't make sense to me - specifically, you always want to use the `venv` pip.

Typically, you use these `Makefile` parameters to be able to configure things. For example, the Python binary used for bootstrapping the `venv` is `python3`, however, on my system `python3 --version` is 3.11, so I can use this `Makefile` by running:
```
make install PYTHON=python3.10
```

Reply:
fine with me.